lib/openscap_parser/datastream_file.rb

```diff
@@ -3,14 +3,10 @@
 require 'openscap_parser/benchmarks'
 
 module OpenscapParser
-  class Datastream
-    include OpenscapParser::XmlFile
+  # A class to represent a datastream (-ds.xml) XmlFile
+  class DatastreamFile < XmlFile
     include OpenscapParser::Benchmarks
 
-    def initialize(report)
-      parsed_xml report
-    end
-
     def valid?
       return true if @parsed_xml.root.name == 'data-stream-collection' && namespaces.keys.include?('xmlns:ds')
       return true if @parsed_xml.root.name == 'Tailoring' && namespaces.keys.include?('xmlns:xccdf')
```

Comment:
I think you can remove this line because tailoring file is now a separate file.

Reply:
Moved to `tailoring_file.rb`